distribution/lib/Standard/Base/0.0.0-dev/src/Metadata.enso

```diff
@@ -5,10 +5,10 @@ import project.Data.Pair.Pair
 import project.Data.Text.Text
 import project.Data.Vector.Vector
 import project.Meta
-import project.Nothing.Nothing
 from project.Data.Boolean import Boolean, False, True
 from project.Data.Json.Extensions import all
 from project.Data.Range.Extensions import all
+from project.Nothing import all
```

Comment:
Isn't this discouraged for non-extension modules?

Reply:
Nothing is now full of extensions.